assets/embauche.html

```diff
@@ -186,9 +178,41 @@
 				<th>Taxe d'apprentissage</th>
 				<td><span data-source="taxe_apprentissage"></span>&nbsp;€</td>
 			</tr>
-			<tr>
+			<tr class="imprecise">
 				<th>Taxe sur les salaires</th>
-				<td><span data-source="taxe_salaires"></span>&nbsp;€</td>
+				<td>
+					<span data-source="taxe_salaires"></span>&nbsp;€
+					<aside>
+						<p>Non calculée. Due par les entreprises non assujetties à la TVA.</p>
+						<a href="https://www.service-public.fr/professionnels-entreprises/vosdroits/F22576" target="_blank">En savoir plus</a>
+					</aside>
+				</td>
+			</tr>
+			<tr class="imprecise">
+				<th>Accident du travail</th>
+				<td>
+					<span data-source="accident_du_travail"></span>&nbsp;€
+					<aside>
+						<p>Montant calculé pour une activité à faible risque d'accident de travail.</p>
+						<a href="http://www.ameli.fr/employeurs/vos-cotisations/le-calcul-des-taux/les-differents-modes-de-tarification_rhone.php" target="_blank">En savoir plus</a>
+					</aside>
+				</td>
+			</tr>
+			<tr class="imprecise">
+				<th>Contribution supplémentaire à l'apprentissage</th>
+				<td>
+					<span data-source="contribution_developpement_apprentissage"></span>&nbsp;€
+					<aside>
+						<p>Calcul imprécis. Cette contribution décroît en fonction du pourcentage de l'effectif en apprentissage.</p>
```

Comment:
Above it's “Non calculée.” (the cotisation). Then it's “Montant calculé”. Here it's “Calcul imprécis”. Make a choice and stick to it  ;)